src/searchForm/SearchForm.style.tsx

```diff
@@ -14,8 +14,7 @@ const formWidth = '928px'
 const formHeight = '56px'
 const primaryFieldsWidth = '240px'
 const secondaryFieldsWidth = '160px'
-const buttonHeight = '48px'
-const buttonHeightSmall = '40px'
```

Comment:
After looking at the designs specs in Figma, both small and large height are 40px.